Data/Infrastructure/OJS.Data.Infrastructure/Extensions/DbContextOptionsBuilderExtensions.cs

```diff
@@ -0,0 +1,5 @@
+
+            return options;
```

Comment:
What is the purpose of this?

Reply:
I can't find a file with that name locally.